log-manager/src/main/java/io/airlift/log/OutputStreamHandler.java

```diff
@@ -0,0 +1,69 @@
+package io.airlift.log;
+
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+
+import static com.google.common.base.Charsets.UTF_8;
+import static java.util.logging.ErrorManager.CLOSE_FAILURE;
+import static java.util.logging.ErrorManager.FLUSH_FAILURE;
+import static java.util.logging.ErrorManager.FORMAT_FAILURE;
+import static java.util.logging.ErrorManager.WRITE_FAILURE;
+
+final class OutputStreamHandler
+        extends Handler
+{
+    private final Writer writer;
+
+    public OutputStreamHandler(OutputStream out)
+    {
+        writer = new OutputStreamWriter(out, UTF_8);
+        setFormatter(new StaticFormatter());
+    }
+
+    @Override
+    public void publish(LogRecord record)
+    {
+        if (!isLoggable(record)) {
+            return;
+        }
+
+        String message;
+        try {
+            message = getFormatter().format(record);
+        } catch (Exception e) {
```

Comment:
This should be `RuntimeException`. `format()` doesn't throw any checked exceptions.